num = 0
try:
    value = 120/num
    print(value)
except:
    print("エラーになりました。")
finally:
    print("計算終わり。")